src/components/Dashboard/TodoList.vue

```diff
@@ -656,46 +704,45 @@ export default {
       }
     },
 
-    async loadAnnualReport (task) {
-      let date
+    /**
+     * Loads a DRAFT/PENDING/ERROR/PAID Annual Report filing.
+     * (Currently used for Coop ARs only, as BComps can't save draft ARs atm.)
+     */
+    async loadAnnualReportFiling (task) {
       const filing: FilingIF = task.task.filing
-      // NB: AR page requires "filing.annualReport"
+      // verify both "header" and "annualReport"
       if (filing?.header && filing?.annualReport) {
-        filing.annualReport.annualReportDate
-          ? date = filing.annualReport.annualReportDate
-          : date = filing.annualReport.nextARDate
-
+        // FUTURE: delete fallback when all draft ARs contain ARFilingYear
+        const ARFilingYear: number =
+          filing.header.ARFilingYear || filing.annualReport.annualReportDate?.substring(0, 4)
         const paymentStatusCode = filing.header.paymentStatusCode
         const payErrorObj = paymentStatusCode ? await this.getPayErrorObj(paymentStatusCode) : null
 
-        if (date) {
```

Comment:
This is always set, so I just removed the superfluous check.

(I tested this in Dev and Test, ie, before and after code changes.)